tox.ini

```diff
@@ -7,7 +7,7 @@ recreate=
 deps=
     -rrequirements.txt
 commands=
-    py.test -v pyquil/tests
+    py.test --reruns 2 -v pyquil/tests
```

Comment:
Kinda confusing that the flag `--reruns` includes the initial run... but that's on them not on you.

Reply:
Good spot! I can see the argument for this behaviour: the first run is not a rerun.

Reply:
Either way, this is awesome. Well done.

Reply:
I was inadvertently thinking about this over a nice bowl of ramen noodles, and maybe the way to go is not to specify the command line flag here and rerun any test that fails, but to instead use the `@pytest.mark.flaky` decorator provided by the library to just mark individual flaky tests the first time they fail.  That has the downside of requiring a mildly annoying manual intervention step, but has the benefit of leaving a marker in the source code of exactly which tests are causing problems, and possibly even goading us into fixing them in the future (a man can dream!).